resources/mixins/kubernetes/mixin.libsonnet

```diff
@@ -9,7 +9,7 @@ kubernetes {
     kubeApiserverSelector: 'job="api"',
     kubeProxySelector: 'job="machine-config-daemon"',
     kubeSchedulerSelector: 'job="scheduler"',
-    namespaceSelector: 'namespace!~"openshift-kube.*|kube.*"'
+    namespaceSelector: 'namespace!~"openshift-kube.*|openshift-marketplace|kube.*"'
```

Comment:
Looking over the `openshift-*` namespaces, should we just ignore everything except maybe `openshift-monitoring`, `openshift-authentication`, `openshift-ingress` and `openshift-logging`? Those are probably the most important ones for us that we would care about even if OSD manages them.

Reply:
Hmm, I guess we could do that. I'm wondering whether we are now at a point where we can switch to a allowlist instead of a deny list after [the initial discussion here.](https://github.com/stackrox/rhacs-observability-resources/pull/108#discussion_r1237980359)

Looking at prod, the most relevant namespaces seem to be:
```
acscs-dataplane-cd
openshift-authentication
openshift-authentication-operator
openshift-ingress
openshift-ingress-canary
openshift-ingress-operator
openshift-logging
openshift-monitoring
rhacs
rhacs-<id>
rhacs-cloudwatch
rhacs-observability
rhacs-probe
```

which would be included by `namespaceSelector: 'namespace~"acscs.*|openshift-authentication.*|openshift-ingress.*|openshift-logging|openshift-monitoring|rhacs.*"'`.